Backpack/src/main/java/net/skyscanner/backpack/calendar/presenter/BpkCalendarController.kt

```diff
@@ -32,6 +33,8 @@ abstract class BpkCalendarController {
 
   internal val selectedRange: CalendarRange = CalendarRange()
 
+  internal var updateCallbackContentCallback: CalendarUpdateCallback? = null
```

Comment:
Can we call this just `updateContentCallback`?

Reply:
🤦‍♂️